src/ir_def/ir.ml

```diff
@@ -158,6 +158,8 @@ and prim =
   (* backend stuff *)
   | CPSAwait of Type.typ
   | CPSAsync of Type.typ
+  | ICPerformGC
+  | ICStableVarsSize
```

Comment:
```suggestion
  | ICStableSize of Type.typ
```
For uniformity, I'd add a type argument, drop Var, and move to the end of the datatype, next to StableWrite and StableSize. Then you don't need to use the argument to infer the type later.